runtime/src/main/scala/akka/grpc/scaladsl/ServiceHandler.scala

```diff
@@ -31,7 +31,7 @@ object ServiceHandler {
       handlers: PartialFunction[HttpRequest, Future[HttpResponse]]*): HttpRequest => Future[HttpResponse] =
     concat(handlers: _*).orElse { case _ => notFound }
 
-  private[scaladsl] def concat(handlers: PartialFunction[HttpRequest, Future[HttpResponse]]*)
+  def concat(handlers: PartialFunction[HttpRequest, Future[HttpResponse]]*)
```

Comment:
I agree it would be useful to open this up, your use case is very sensible.

When we open this up, perhaps we should reconsider whether 'varargs' is the way to go here, or that it would be better to explicitly expect a collection, such as `immutable.Seq`. What do you think?

Reply:
Would you want that to be a change to concatOrNotFound too?

Reply:
Ah, makes sense to keep those consistent, so this is :+1: